src/components/CalendarContainer.tsx

```diff
@@ -73,6 +73,7 @@ export interface CalendarContainerProps<T> {
   onPressEvent?: (event: ICalendarEvent<T>) => void
   weekEndsOn?: WeekNum
   maxVisibleEventCount?: number
+  eventMinHeight?: number
```

Comment:
Can you rename it to `eventMinHeightForMonthView` as it affects only for the month view?